svc-bip-api/src/main/java/gov/va/vro/bip/config/RabbitMqConfig.java

```diff
@@ -46,4 +57,47 @@ RabbitListenerErrorHandler svcBipApiErrorHandler() {
         };
     return handler;
   }
+
+  @Bean
+  RabbitListenerErrorHandler svcBipApiErrorHandlerV2(ObjectMapper mapper) {
```

Comment:
What's the purpose of having two separate error handler versions? When should I use one over the other?

Reply:
The old one will be refactored out of existence when the refactors are completed for the remaining endpoints.